src/App.test.tsx

```diff
@@ -0,0 +1,18 @@
+import { describe, expect, it } from 'vitest';
+import { render, screen } from '@testing-library/react';
+
+import App from './App';
+
+describe('App', () => {
+  it('Renders heading', () => {
+    //Arrange
+    render(<App />);
+    //Act
+    //Expect
+    expect(
+      screen.getByRole('heading', {
```

Comment:
The test is failing with getByRole. You have async actions in the component. In such case you should use findByRole

Reply:
Working on tests in PR #3